recipes/pygraphml/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "pygraphml" %}
+{% set version = "2.2" %}
+{% set sha256 = "cbc3aedcf9b2d3c7c7c47c75182588b334ee3cb6ad0df047d2b368cdb0a8dd1b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - pygraphml
+
+about:
+  home: https://github.com/hadim/pygraphml/
+  license: BSD License
```

Comment:
[It looks](https://github.com/hadim/pygraphml/blob/master/LICENSE) like a `BSD 3-Clause`. Can you update the license version here before we merge this?